pkg/util/logutil/log.go

```diff
@@ -0,0 +1,7 @@
+package logutil
```

Comment:
I'd suggest not repeating "util" suffix in these package names, and only use non-conflicting names on imports.

Reply:
What name would you suggest in this case?

Reply:
`pkg/util/log`, and import using `import util_log "github.com/cortexproject/cortex/pkg/util/log"`.